src/test/java/apicurito/tests/utils/slenide/CommonUtils.java

```diff
@@ -51,6 +52,36 @@ public static SelenideElement getNewPlusSignButton(Sections section, SelenideEle
         return differentRoot.$$("div").filter(attribute("class", section.getSectionElementName())).first().$("button");
     }
 
+    public static void setValueInTextArea(String value, SelenideElement section){
+        log.info("Setting value {} in text area in section{}", value, section.getValue());
+
+        section.$$("div").filter(attribute("title", "Click to edit.")).first().click();
+
+        section.$("ace-editor textarea").sendKeys(Keys.CONTROL + "a");
+        section.$("ace-editor textarea").sendKeys(Keys.DELETE);
+        section.$("ace-editor textarea").sendKeys(value);
+
+        getButtonWithTitle("Save changes.", section).click();
+    }
+
+    public static void setValueInLabel(String value, SelenideElement section, Boolean isPencilLabel){
```

Comment:
Any particular reason this is Boolean instead of boolean?

Reply:
no reason --> changed